diff --git a/xorg-server/installer/vcxsrv-64-debug.nsi b/xorg-server/installer/vcxsrv-64-debug.nsi
index b346bce45..04874f587 100644
--- a/xorg-server/installer/vcxsrv-64-debug.nsi
+++ b/xorg-server/installer/vcxsrv-64-debug.nsi
@@ -26,10 +26,10 @@
 Name "${NAME}"
 
 ; The file to write
-OutFile "vcxsrv-64-debug.${VERSION}.installer.exe"
+OutFile "vcxsrv-64-debug.${VERSION}.installer.noadmin.exe"
 
 ; The default installation directory
-InstallDir $programfiles64\VcXsrv
+InstallDir C:\VcXsrv
 
 SetCompressor /SOLID lzma
 
@@ -45,8 +45,8 @@ VIAddVersionKey /LANG=${LANG_ENGLISH} "FileDescription" "VcXsrv windows xserver"
 VIAddVersionKey /LANG=${LANG_ENGLISH} "FileVersion" "${VERSION}"
 VIAddVersionKey /LANG=${LANG_ENGLISH} "ProductVersion" "${VERSION}"
 
-; Request application privileges for Windows Vista
-RequestExecutionLevel admin
+; Request application privileges for user
+RequestExecutionLevel user
 
 ;--------------------------------
 InstType "Full"
@@ -71,7 +71,7 @@ XPStyle on
 ; The stuff to install
 Section "VcXsrv debug exe and dlls"
 
-  SetShellVarContext All
+  SetShellVarContext current
 
   SectionIn RO
   SectionIn 1 2 3
@@ -195,7 +195,7 @@ SectionEnd
 Section "Fonts"
   SectionIn 1 3
 
-  SetShellVarContext All
+  SetShellVarContext current
 
   SetRegView 64
 
@@ -209,7 +209,7 @@ SectionEnd
 Section "Start Menu Shortcuts"
   SectionIn 1 3
 
-  SetShellVarContext All
+  SetShellVarContext current
 
   SetRegView 64
 
@@ -223,7 +223,7 @@ SectionEnd
 Section "Desktop Shortcuts"
   SectionIn 1 3
 
-  SetShellVarContext All
+  SetShellVarContext current
 
   SetRegView 64
 
@@ -240,7 +240,7 @@ Section "Uninstall"
 
   SetRegView 64
 
-  SetShellVarContext All
+  SetShellVarContext current
 
   ; Remove registry keys
   DeleteRegKey HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\VcXsrv"
diff --git a/xorg-server/installer/vcxsrv-64.nsi b/xorg-server/installer/vcxsrv-64.nsi
index c213f5415..f5d1e151a 100644
--- a/xorg-server/installer/vcxsrv-64.nsi
+++ b/xorg-server/installer/vcxsrv-64.nsi
@@ -27,10 +27,10 @@
 Name "${NAME}"
 
 ; The file to write
-OutFile "vcxsrv-64.${VERSION}.installer.exe"
+OutFile "vcxsrv-64.${VERSION}.installer.noadmin.exe"
 
 ; The default installation directory
-InstallDir $programfiles64\VcXsrv
+InstallDir C:\VcXsrv
 
 SetCompressor /SOLID lzma
 
@@ -46,8 +46,8 @@ VIAddVersionKey /LANG=${LANG_ENGLISH} "FileDescription" "VcXsrv windows xserver"
 VIAddVersionKey /LANG=${LANG_ENGLISH} "FileVersion" "${VERSION}"
 VIAddVersionKey /LANG=${LANG_ENGLISH} "ProductVersion" "${VERSION}"
 
-; Request application privileges for Windows Vista
-RequestExecutionLevel admin
+; Request user privileges
+RequestExecutionLevel user
 
 ;--------------------------------
 InstType "Full"
@@ -72,7 +72,7 @@ XPStyle on
 ; The stuff to install
 Section "VcXsrv (required)"
 
-  SetShellVarContext All
+  SetShellVarContext current
 
   SectionIn RO
   SectionIn 1 2 3
@@ -196,7 +196,7 @@ SectionEnd
 Section "Fonts"
   SectionIn 1 3
 
-  SetShellVarContext All
+  SetShellVarContext current
 
   SetRegView 64
 
@@ -210,7 +210,7 @@ SectionEnd
 Section "Start Menu Shortcuts"
   SectionIn 1 3
 
-  SetShellVarContext All
+  SetShellVarContext current
 
   SetRegView 64
 
@@ -224,7 +224,7 @@ SectionEnd
 Section "Desktop Shortcuts"
   SectionIn 1 3
 
-  SetShellVarContext All
+  SetShellVarContext current
 
   SetRegView 64
 
@@ -241,7 +241,7 @@ Section "Uninstall"
 
   SetRegView 64
 
-  SetShellVarContext All
+  SetShellVarContext current
 
   ; Remove registry keys
   DeleteRegKey HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\VcXsrv"
diff --git a/xorg-server/installer/vcxsrv-debug.nsi b/xorg-server/installer/vcxsrv-debug.nsi
index b0e6d103a..81e708974 100644
--- a/xorg-server/installer/vcxsrv-debug.nsi
+++ b/xorg-server/installer/vcxsrv-debug.nsi
@@ -23,13 +23,13 @@
 Name "${NAME}"
 
 ; The file to write
-OutFile "vcxsrv-debug.${VERSION}.installer.exe"
+OutFile "vcxsrv-debug.${VERSION}.installer.noadmin.exe"
 
 ; The default installation directory
-InstallDir $PROGRAMFILES32\VcXsrv
+InstallDir C:\VcXsrv
 
 Function .onInit
-StrCpy $InstDir $PROGRAMFILES32\VcXsrv
+StrCpy $InstDir C:\VcXsrv
 FunctionEnd
 
 SetCompressor /SOLID lzma
@@ -46,8 +46,8 @@ VIAddVersionKey /LANG=${LANG_ENGLISH} "FileDescription" "VcXsrv windows xserver"
 VIAddVersionKey /LANG=${LANG_ENGLISH} "FileVersion" "${VERSION}"
 VIAddVersionKey /LANG=${LANG_ENGLISH} "ProductVersion" "${VERSION}"
 
-; Request application privileges for Windows Vista
-RequestExecutionLevel admin
+; Request application privileges for user
+RequestExecutionLevel user
 
 ;--------------------------------
 InstType "Full"
diff --git a/xorg-server/installer/vcxsrv.nsi b/xorg-server/installer/vcxsrv.nsi
index 819ef7abc..c56033100 100644
--- a/xorg-server/installer/vcxsrv.nsi
+++ b/xorg-server/installer/vcxsrv.nsi
@@ -27,13 +27,13 @@
 Name "${NAME}"
 
 ; The file to write
-OutFile "vcxsrv.${VERSION}.installer.exe"
+OutFile "vcxsrv.${VERSION}.installer.noadmin.exe"
 
 ; The default installation directory
-InstallDir $PROGRAMFILES32\VcXsrv
+InstallDir C:\VcXsrv
 
 Function .onInit
-StrCpy $InstDir $PROGRAMFILES32\VcXsrv
+StrCpy $InstDir C:\VcXsrv
 FunctionEnd
 
 SetCompressor /SOLID lzma
@@ -50,8 +50,8 @@ VIAddVersionKey /LANG=${LANG_ENGLISH} "FileDescription" "VcXsrv windows xserver"
 VIAddVersionKey /LANG=${LANG_ENGLISH} "FileVersion" "${VERSION}"
 VIAddVersionKey /LANG=${LANG_ENGLISH} "ProductVersion" "${VERSION}"
 
-; Request application privileges for Windows Vista
-RequestExecutionLevel admin
+; Request application privileges for user
+RequestExecutionLevel user
 
 ;--------------------------------
 InstType "Full"
@@ -76,7 +76,7 @@ XPStyle on
 ; The stuff to install
 Section "VcXsrv (required)"
 
-  SetShellVarContext All
+  SetShellVarContext current
 
   SectionIn RO
   SectionIn 1 2 3
@@ -197,7 +197,7 @@ SectionEnd
 Section "Fonts"
   SectionIn 1 3
 
-  SetShellVarContext All
+  SetShellVarContext current
 
   SetOutPath $INSTDIR\fonts
   CreateDirectory "$INSTDIR\VcXsrv"
@@ -209,7 +209,7 @@ SectionEnd
 Section "Start Menu Shortcuts"
   SectionIn 1 3
 
-  SetShellVarContext All
+  SetShellVarContext current
 
   SetOutPath $INSTDIR
   CreateShortCut "$INSTDIR\Uninstall VcXsrv.lnk" "$INSTDIR\uninstall.exe" "" "$INSTDIR\uninstall.exe" 0
@@ -221,7 +221,7 @@ SectionEnd
 Section "Desktop Shortcuts"
   SectionIn 1 3
 
-  SetShellVarContext All
+  SetShellVarContext current
 
   SetOutPath $INSTDIR
   CreateShortCut "$DESKTOP\XLaunch.lnk" "$INSTDIR\xlaunch.exe" "" "$INSTDIR\xlaunch.exe" 0
@@ -234,7 +234,7 @@ SectionEnd
 
 Section "Uninstall"
 
-  SetShellVarContext All
+  SetShellVarContext current
 
   ; Remove registry keys
   DeleteRegKey HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\VcXsrv"
