diff --git a/tools/skqp/download_model.py b/tools/skqp/download_model.py
index fb0020e481..a5d8a03754 100755
--- a/tools/skqp/download_model.py
+++ b/tools/skqp/download_model.py
@@ -10,7 +10,7 @@ import os
 import shutil
 import sys
 import tempfile
-import urllib2
+import urllib.request, urllib.error, urllib.parse

 def checksum(path):
     if not os.path.exists(path):
@@ -33,7 +33,7 @@ def download(md5, path):
                 pass  # ignore race condition
         url = 'https://storage.googleapis.com/skia-skqp-assets/' + md5
         with open(path, 'wb') as o:
-            shutil.copyfileobj(urllib2.urlopen(url), o)
+            shutil.copyfileobj(urllib.request.urlopen(url), o)

 def tmp(prefix):
     fd, path = tempfile.mkstemp(prefix=prefix)
